_posts/2018-02-27-what-should-go-stdlib.md

```diff
@@ -0,0 +1,216 @@
+---
+title: "What Should Go Into the C++ Standard Library"
+layout: blog
+sidenav: side-nav-blog.html
+published: false
+permalink: blog/20180227-what-should-go-stdlib
+type: markdown
+---
+
+By [Titus Winters](mailto:titus@google.com), @TitusWinters
```

Comment:
Nit:

For the tips, we've using the format:

By Titus Winters ([titus@google.com](mailto:titus@google.com))

No big deal, but I don't see a reason to change it right now.

Reply:
Done